src/main/java/bit/javaoop/FixedEmployeeNumberManager.java

```diff
@@ -0,0 +1,15 @@
+package bit.javaoop;
+
+public class FixedEmployeeNumberManager extends Manager {
+
+    public FixedEmployeeNumberManager(String name, double salary, int counter, int employeers) {
+        super(name, salary, counter);
+        Employee tab[] = new Employee[employeers];
+        setTab(tab);
+    }
+    @Override
+    public boolean isSatisfied(){
+        return getSalary() > 20000 && (getCounter() == getTab().length);
```

Comment:
Takie magiczne liczby jak te 20000 można wyekstrahować do stałej i nazwać ( skrót ctrl + alt + C) np. w ten sposób
![image](https://user-images.githubusercontent.com/18233796/37981175-324c173e-31ed-11e8-8388-bcbcd05911b8.png)
lepiej się to wtedy czyta:)